libs/kg-commons/src/main/java/eu/ebrains/kg/commons/jsonld/DynamicJson.java

```diff
@@ -130,6 +131,14 @@ public static boolean isInternalKey(String key) {
         return key.startsWith("_");
     }
 
+    public static boolean isNotNecessaryKey(String key) {
```

Comment:
The logic is ok - but the DynamicJson is supposed to be more generic. With the new functions, you're introducing EBRAINS logic into the class. Couldn't we move those to the "NormalizedJsonLd" class?